### PR TITLE
Add Reopen Clinical Submission Mutation

### PR DESCRIPTION
**Pull request by jaserud**
**Description of changes**

- added support for reopen
- also removed .zip from all template endpoint/path

**Type of Change**

- [x] Bug
- [ ] Styling
- [ ] New Feature

### CODE REVIEW

File: routes/clinical.js [hunk 1]
```diff
@@ -10,7 +10,9 @@ var router = express.Router();
 // so that the user is automatically prompted to save the file from their browser.
 const apiRoot = urlJoin(CLINICAL_SERVICE_ROOT, SUBMISSION_TEMPLATE_PATH);
 
-// This is mostly passthrough, so file names and response should be set in clincial
+// This is mostly passthrough, so file names and response will be set in clincial
```
Comment (jaserud) [inline routes/clinical.js]: @hlminh2000 not sure if this affects you right now...

Comment (hlminh2000) [reply routes/clinical.js]: sounds backward compatible so shouldn't be a breaking change for me. Though does this explanation appear in swagger somewhere? :P

Comment (jaserud) [reply routes/clinical.js]: no lol, guess I could put in something

Comment (hlminh2000) [reply routes/clinical.js]: yeah I just figured as a consumer of the API I'd prefer to find this information there than digging into source code :P thanks!

Comment (jaserud) [reply routes/clinical.js]: I removed the all.zip it was more confusing

**Pull request merged**